src/store/reducer.ts

```diff
@@ -46,6 +48,9 @@ const initialState: State = {
   offers: [],
   authentication: {
     timestamp: 0
+  },
+  userSettings: {
+    preferredCurrencyCode: baseCurrencyCode
```

Comment:
maybe `initialCurrencyCode` instead of `baseCurrencyCode`, alhtough I am not sure